packages/test/mocha-test-setup/src/mochaHooks.ts

```diff
@@ -37,7 +37,13 @@ class TestLogger implements ITelemetryBufferedLogger {
 		event.testName = this.testName ?? currentTestName;
 		event.testVariant = testVariant;
 		event.hostName = pkgName;
-		this.parentLogger.send({ ...event, ...propsDict });
+		this.parentLogger.send({
+			...event,
+			...propsDict,
+			details: JSON.stringify({
+				displayName: process.env.FLUID_TEST_PIPELINE_IDENTIFIER ?? "",
+			}),
+		});
```

Comment:
I just realized we are taking that `propsDict` from an env variable already (line 21 above), and we even use it in one place... [to override the hostName](https://github.com/microsoft/FluidFramework/blob/c4560a7e7469be32ae66cff60b4d6f1c292c65d8/tools/pipelines/test-perf-benchmarks.yml#L604) 😄 . Maybe instead of adding the new variable, we just leverage the existing one? I'd even be tempted to put everything that came from `FLUID_LOGGER_PROPS` into `details:` for the event, and maybe for now add a special case so if there's a `hostName` property, we still override the one outside `details` (so we can update the current use of `FLUID_LOGGER_PROPS` to have both `hostname: Benchmark` and `displayName: ${{ variables.pipelineIdentifierForTelemetry }}` in the same JSON object).